api/bags_api/src/main/scala/uk/ac/wellcome/platform/storage/bags/api/BagsApi.scala

```diff
@@ -129,19 +130,14 @@ trait BagsApi extends Logging {
             )
 
           case Right(manifests) =>
-            val etagValue = manifests.map(_.idWithVersion).mkString("&")
-            val etag = ETag(etagValue)
```

Comment:
What happened to ethe tag here?

Reply:
The tests flushed out that we are unlikely to get large multi version responses (as they will likely result from having _many_ versions), and if we do assembling the key like this will result in extremely long and possibly invalid keys.